### PR TITLE
Fix clone view

### PR DESCRIPTION
**Pull request by bijayprasadtiwari**
Replaced the C# based cmdlet with the script based. Now the cmdlet will also support cloning from a inactive view.

### CODE REVIEW

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Copy-CohesityView -TaskName "Task-clone-a-view" -SourceViewName "source-view" -TargetViewName "target-view" -TargetViewDescription "Create a view clone" -QosPolicy "Backup Target Low" -JobId 12345
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$TaskName,
```
Comment (ashish-cohesity) [inline src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:20]: Can we make this optional ? If user doesn't not give this then create a 'Clone-<viewname>-<timestamp>' ?

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$SourceViewName,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$TargetViewName,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$TargetViewDescription,
```
Comment (ashish-cohesity) [inline src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:29]: Why is description mandatory ? can we make this optional please ?

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [long]$JobId,
+        [Parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]
+        [string]$QoSPolicy = "Backup Target Low",
```
Comment (ashish-cohesity) [inline src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:35]: This should be a set to chose from ,  if not specified default it to the above.

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        .SYNOPSIS
+        Clones the specified Cohesity View.
+        .DESCRIPTION
+        Clones the specified Cohesity View.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Copy-CohesityView -TaskName "Task-clone-a-view" -SourceViewName "source-view" -TargetViewName "target-view" -TargetViewDescription "Create a view clone" -QosPolicy "Backup Target Low" -JobId 12345
```
Comment (ashish-cohesity) [inline src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:12]: If backward compatibility is not at stake get rid  of jobId, let the function figure out the backup job and clone view from that one .

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 (hunk 1 above)
Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:20]: Fixed

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 (hunk 2 above)
Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:29]: Fixed

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 (hunk 3 above)
Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:35]: Fixed

File: src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1 (hunk 4 above)
Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1:12]: The user need to specify the job id to create a clone. The view could be cloned by multiple jobs. The parameters are kept unchanged from the C# based cmdlet.

**Pull request merged**